README.md

```diff
@@ -121,16 +121,16 @@ Returns a promise that resolves into the payload in the form of a Buffer or (opt
 - **options** - A configuration object
     - **timeout** - The number of milliseconds to wait while reading data before
     aborting handling of the response. Defaults to unlimited.
-    - **json** - A value indicating how to try to parse the payload as JSON. Defaults to `true`.
+    - **json** - A value indicating how to try to parse the payload as JSON. Defaults to `false`.
```

Comment:
No JSON parsing has always been the default case for Wreck, they just more recently added the explicit `false` option instead of just passing `undefined`. We'll want to [continue defaulting it to `true`](https://github.com/ExpediaGroup/service-client/blob/master/lib/schema.js#L60) in service-client.